internal/crossdock/client/errorshttpclient/behavior.go

```diff
@@ -150,8 +150,7 @@ func Run(t crossdock.T) {
 			},
 			body:       `{"token":"10"}`,
 			wantStatus: 501,
-			wantBody: `unrecognized procedure ` +
-				`"echo" for service "not-yarpc-test"` + "\n",
+			wantBody:   `unrecognized service name "not-yarpc-test"` + "\n",
```

Comment:
nit: no need for the `+`

Reply:
I got `Not equal: "unrecognized service name \"not-yarpc-test\"\\n" (expected) 
!= "unrecognized service name \"not-yarpc-test\"\n" (actual)` after change original message to `unrecognized service name "not-yarpc-test"\n`. I guess `+ "\n"` is used to prevent escape characters.